test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -125,13 +132,11 @@ private static void checkTransferredContents(InputStreamProvider inputStreamProv
             int count = inBytes.length - posIn;
 
             if (reported != count)
-                throw new AssertionError(
-                        format("reported %d bytes but should report %d", reported, count));
+                fail(format("reported %d bytes but should report %d", reported, count));
```

Comment:
I would consider using

 assertTrue( reported == count, format("reported %d bytes but should report %d", reported, count));

Reply:
Done in https://github.com/openjdk/jdk/pull/5421/commits/b37ad6b39ffc78af7ca8c487125b7e4e0c1ff141.